### ISSUE TITLE
Instruction Text Box

### ISSUE THREAD

**Issue by Fourniture** [musescore/MuseScore#25555]
### Your idea

When writing music for organ, it is very common to have an instruction box below the title detailing the stops the performer is to use. 
![image](https://github.com/user-attachments/assets/74dee846-07a2-4f24-912a-e6539ced5410)
I would like a specific option in the Add -> Text menu, called "Instructions". This would be another text box that appears directly under the title box. 
   In addition to this, it would be very pleasant to have an option in preferences that automatically adds this instruction box to new scores if organ is the only instrument in the score.

### Problem to be solved

There is no dedicated option for this.

### Additional context

This box would be titled instructions, because it is really meant for any instructions to a player for preparing their instrument.

### Checklist

- [X] This request follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting issues
- [X] I have verified that this feature request has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar requests

**Comment by bkunda** [comment]
This can already be achieved with the existing text boxes. We wouldn't add a special kind of text box specifically for organ music (if we were to do this for every instrument, the program would become unworkable).

Is there something special about organ stops you are requesting here?

**Comment by Fourniture** [comment]
No, I only would like a specific option for a text box. This would be a text box that appears directly under the title, called "Instruction" or "Performance Instructions". I consider it necessary to have a special option for this because it is distinct from a title, footer, header, etc., and it has a specific spot it should appear. It also is not just for organ, but also for any instruments where special instructions are needed for preparing them (i.e. prepared pianos, harp/strings scordatura, etc.

**Comment by MarcSabatella** [comment]
But how would this text box differ from the ordinary text box that already exists and you can already place directly under the title?

**Comment by Fourniture** [comment]
The point is not that something should differ to make it worth putting in as a feature; the point is that new features should improve workflows, ease of use, or show something that the user wants, even if the requested feature already exists through another broader feature. A user wants to see the most direct way to do something: with a dedicated feature. 
   An instruction box (needed often) could be added with an ordinary text box, but so could a title, copyright and more. This fact does not make a dedicated title or copyright field unnecessary.

**Comment by MarcSabatella** [comment]
But there *aren't* separate frame types for any of those things!  Titles sit within perfectly ordinary vertical frames, just as instrument instructions can.  Copyright messages within the footer right alongside any other information you wish to place there.  There are specific *text styles* for these and other particular types of text, because the *formatting* of these differs from that or more ordinary text.  Titles are different from ordinary text in that they are usually larger and centered, and often bold.  Composer text is different in that it is right and bottom aligned within the frame.  And so on.  But again, there aren't special dedicated frame types for any of these things, nor is there any reason there would need to be.

So if you're just saying, you want a special dedicate text style for instrument instructions, the relevant question to answer is, how would the formatting need to differ from ordinary frame text?

**Comment by Fourniture** [comment]
I am asking for an option for a generic text frame that is labeled in the menu as "Instruction Frame". Yes, it is the same as an ordinary text frame, but it counts as a different option. For a broader discussion of this idea, please see issue #25589
